docs/source/operations/projections/merc.rst

```diff
@@ -6,7 +6,10 @@ Mercator
 
 The Mercator projection is a cylindrical map projection that origins from the 15th
```

Comment:
In an (admittedly modest) search, I have not found evidence for 15th century traces of Mercator's projection. He was, himself born 1512, and his first map using this projection appears to date from 1569. Hence, this should probably read "from the 16th century" (although, I'm sure, being liberal enough with what one consider origins we may find traces back to any century we'd like)

Reply:
Yes, that should be 16th cent.